### DISCUSSION TITLE
How to Create a VictoryCandlestick Chart in Victory-Native XL?

### DISCUSSION THREAD

**Discussion by auttaponshine** [FormidableLabs/victory-native-xl#354] [Q&A]
Thanks for this awesome repo.

I am trying to create a VictoryCandlestick chart using victory-native (XL) in a React Native project. I saw examples for web with victory but haven't found much documentation specifically for victory-native when working with VictoryCandlestick.

Could someone guide me on how to correctly implement VictoryCandlestick using victory-native (XL), especially with larger datasets (Victory XL)? Are there any best practices or performance tips for handling large candlestick charts in React Native? Any working example or snippet would be really helpful!

Thanks in advance!

**Comment by DieterEOnofre** [comment]
Did you get any further with your request? Im trying to do the same thing.

**Comment by Miloshinjo** [comment]
Same thing here, I tried many options but could not do it. Is the solution for now to use the old library?

**Comment by ghost** [comment]
Necro-ing this discussion just to show interest. Building both line and candlestick charts atm and exploring all options so definitely +1 from my side.
